packages/portal/src/components/generic/ConfirmDangerModal.vue

```diff
@@ -61,6 +62,23 @@
       promptText: {
         type: String,
         default: null
+      },
+
+      value: {
+        type: Boolean,
+        default: false
+      }
+    },
+
+    data() {
+      return {
+        show: this.value
+      };
+    },
+
+    watch: {
+      value() {
```

Comment:
Does this ever get called, now the the ConfirmDangerModal instancces always have the same value for v-if and v-model?